### PR TITLE
Use latest MCE version

### PR DESCRIPTION
**Pull request by ellatrix**
This fixes both #2493 and #2494. Something in the nightly version of TinyMCE is causing these bugs.

1. @EphoxJames @tg-ephox @spocke Any ideas? :)
2. I really think we should do this change regardless of a nightly fix. It's really hard to track these bugs down without any commit making this change. I'd rather bump the version each time a new MCE version is released, or pin it to a specific commit. Cc @nylen @aduth.

@mtias Does this fix the Sublime paste issue you were having as well?

### CODE REVIEW

**Comment by iseulde** [comment]
@nylen @codecov These reports are still so strange... How does this affect test coverage at all?

**Review by youknowriad — APPROVED** [review]
Sounds good to me 👍 Now that the link boundaries are on the latest

**Comment by iseulde** [comment]
Okay, let's merge. :)

File: lib/client-assets.php [hunk 1]
```diff
@@ -204,23 +204,23 @@ function gutenberg_register_vendor_scripts() {
...
+		'tinymce-latest-paste',
+		'https://fiddle.azurewebsites.net/tinymce/4.6.5/plugins/paste/plugin' . $suffix . '.js',
+		array( 'tinymce-latest' )
 	);
 	gutenberg_register_vendor_script(
-		'tinymce-nightly-table',
-		'https://fiddle.azurewebsites.net/tinymce/nightly/plugins/table/plugin' . $suffix . '.js',
-		array( 'tinymce-nightly' )
+		'tinymce-latest-table',
+		'https://fiddle.azurewebsites.net/tinymce/4.6.5/plugins/table/plugin' . $suffix . '.js',
```
Comment (EphoxJames) [inline lib/client-assets.php]: Only thing I would have done differently is put the version number in one place so you can update it in one place.

Comment (iseulde) [inline lib/client-assets.php]: Feel free :)

Comment (EphoxJames) [inline lib/client-assets.php]: Done:  https://github.com/WordPress/gutenberg/pull/2516

**Pull request merged**